pkg/providers/instancetype/instancetypes.go

```diff
@@ -233,3 +237,49 @@ func MaxEphemeralOSDiskSizeGB(sku *skewer.SKU) float64 {
 	// convert bytes to GB
 	return maxDiskBytes / float64(units.Gigabyte)
 }
+
+var (
+	// https://learn.microsoft.com/en-us/azure/reliability/availability-zones-service-support#azure-regions-with-availability-zone-support
+	// (could also be obtained programmatically)
```

Comment:
Probably makes more sense to add a hack/codegen script for this similar to instance types, right?

Reply:
For production, we could just determine this directly by using the API on startup; only need to do it for the region in use.
For unit testing - Yes, it would be useful to have non-zonal region represented, though codegen would be an overkill, I think.

Reply:
Thinking about this some more - codegen for _instance types_ themselves for a non-zonal region would actually make sense, and may be needed for proper unit test coverage